zxcvbn-cs/README.md

```diff
@@ -101,3 +101,7 @@ The following changes differ from the [original library](https://github.com/mick
 	
 - Change dictionaries load
 	Loads all embedded resources from the directory "zxcvbn.Dictionaries"
```

Comment:
Add translation changes

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/13379cfef9e4d4413c842d1c21901feaddfbcc36

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/5627d9ebe657a3afc2b05c36bce9bd63a9f2e7b6